wagtailmenus/tests/test_disabling_menus_in_cms_settings.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+from django.test import TestCase, override_settings
+from mock import call, patch
+
+from wagtailmenus import wagtail_hooks
+
+try:
+    from importlib import reload  # Python 3.4+
+except ImportError:
+    try:
+        from imp import reload  # Python 3
+    except ImportError:
+        import reload  # Python 2.x
```

Comment:
Wagtailmenus is Python 3 only now, so it should be possible to simplify the imports a little

Reply:
In tox I only see Python 3.4 and up, should Python3 versions lower than 3.4 also be supported?

Reply:
Just 3.4 and up will do :)